source/rust_verify/src/main.rs

```diff
@@ -27,6 +27,15 @@ pub fn main() {
     let internal_program = internal_args.next().unwrap();
     let build_test_mode = if let Some(first_arg) = internal_args.next() {
```

Comment:
`internal_args` is intended for calls to `verus` originated by the build system, or verus itself (for alternative drivers for verifying the standard library, or our mechanism to run the borrow checker only on proof code). Can you please add `version` to the args in https://github.com/verus-lang/verus/blob/2285a9d3769c530ed0461251e82694327ce641de/source/rust_verify/src/config.rs#L32
and parse it from https://github.com/verus-lang/verus/blob/2285a9d3769c530ed0461251e82694327ce641de/source/rust_verify/src/config.rs#L110 instead?